naptan_update/update_table.py

```diff
@@ -47,23 +46,21 @@ def lambda_handler():
         )
     except ClientError as e:
         print(e.response['Error']['Message'])
-        raise
+        raise CannotGetModificationDate(e.response['Error']['Message'])
     else:
-        modification_date = response['Item']['ModificationDate']
+        modification_date = response['Item']['ModificationDateTime']
 
     try:
-        with codecs.open("/tmp/stops_temp_file.csv", 'r', encoding='ascii', errors='ignore') as stops_csv:
+
+        with codecs.open(TMP_FILE_PATH, 'r', encoding='ascii', errors='ignore') as stops_csv:
             reader = csv.DictReader(stops_csv)
             for idx, row in enumerate(reader):
-                # if stop modification date
+                # If stop modification date
                 if row['ModificationDateTime'] >= modification_date:
-                    # todo verify that delete_item is safe to in case item does not exists
-                    # try to delete the item, assuming that if item does not exists, action will just do nothing
-                    table.delete_item(Key={'StopId': row['ATCOCode']})
                     stop = create_stop(row)
-                    # put updated stop in DB
+
+                    # Put updated stop in DB. In case stop already exists this action will overwrite it
                     table.put_item(Item=stop)
```

Comment:
Let's aggregate updates and do them in batch update

Reply:
done